apps/web/src/components/purchase-form/crypto/sections/crypto-form-wc.tsx

```diff
@@ -125,7 +101,7 @@ export default function CryptoFormWalletConnect({
     if (connector) {
       setLoadingText(t('common:statuses.Connected to Wallet'))
       const assetTx = await algorand.makeAssetTransferTransaction({
-        amount: usdcPriceInt * 10_000,
+        amount: price,
```

Comment:
@smonn does this seem right to you? we're transferring usdc here, not micro-algos.

Reply:
This was unfortunately not terribly helpful: [`makeAssetTransferTxnWithSuggestedParamsFromObject`](https://algorand.github.io/js-algorand-sdk/modules.html#makeAssetTransferTxnWithSuggestedParamsFromObject)

Reply:
Short version: assuming `price` here is in the correct denomination, yes this should be fine. 

Slightly longer version: USDCa is defined with 6 decimals ([see TestNet version](https://testnet.algoexplorer.io/asset/10458941) or [MainNet version](https://algoexplorer.io/asset/31566704)). This means `amount` here would be $1 = 1,000,000. It's similar to microAlgos in that sense; when sending a payment in ALGO you specify it as microAlgos where 1 ALGO = 1,000,000 microAlgos. I don't know how/where `price` came from so I can't give a straight answer, but I'd verify that you're providing "microUSDCa" here. So if you're expecting $10, then the value should be 10,000,000.

Reply:
Ah ok! So price here (and everywhere now) is in cents so yeah (price / 100) * 1,000,000 would give us that 10_000 multiplier. I'll put that back in and we should be good to merge.